apache-cassandra-mixin/signals/overview.libsonnet

```diff
@@ -0,0 +1,415 @@
+      unit: 'bytes',
+      sources: {
+        prometheus: {
+          expr: 'sum(cassandra_storage_load_count{%(queriesSelector)s}) by (cassandra_cluster)',
+          legendCustomTemplate: '{{ instance }}',
+        },
+      },
+    },
+
+    diskUsagePanel: {
+      name: 'Disk usage',
+      nameShort: 'Disk usage',
+      type: 'raw',
+      description: 'The disk usage being reported',
+      unit: 'bytes',
+      sources: {
+        prometheus: {
+          expr: 'sum(cassandra_storage_load_count{%(queriesSelector)s}) by (cassandra_cluster, instance)',
+          legendCustomTemplate: '{{ instance }}',
+        },
+      },
+    },
+
+    writes: {
+      name: 'Writes',
+      nameShort: 'Writes',
+      type: 'raw',
+      description: 'The number of writes being reported',
+      unit: 'none',
+      sources: {
+        prometheus: {
+          expr: 'sum by (cassandra_cluster) (increase(cassandra_keyspace_writelatency_seconds_count{%(queriesSelector)s}[$__rate_interval]))',
+        },
+      },
+    },
+
+    reads: {
+      name: 'Reads',
+      nameShort: 'Reads',
+      type: 'raw',
+      description: 'The number of reads being reported',
+      unit: 'none',
+      sources: {
+        prometheus: {
+          expr: 'sum by (cassandra_cluster) (increase(cassandra_keyspace_readlatency_seconds_count{%(queriesSelector)s}[$__rate_interval]))',
+        },
+      },
+    },
+
+    writeAverageLatency: {
+      name: 'Write average latency',
+      nameShort: 'Write average latency',
+      type: 'gauge',
+      description: 'The average write latency being reported',
+      unit: 's',
+      sources: {
+        prometheus: {
+          expr: 'sum(cassandra_keyspace_writelatency_seconds_average{%(queriesSelector)s} >= 0) by (cassandra_cluster)',
+        },
+      },
+    },
+
+    readAverageLatency: {
+      name: 'Read average latency',
+      nameShort: 'Read average latency',
+      type: 'gauge',
+      description: 'The average read latency being reported',
+      unit: 's',
+      sources: {
+        prometheus: {
+          expr: 'sum(cassandra_keyspace_readlatency_seconds_average{%(queriesSelector)s} >= 0) by (cassandra_cluster)',
+        },
+      },
+    },
+
+    writeLatencyHeatmap: {
+      name: 'Write latency',
+      nameShort: 'Write latency',
+      type: 'gauge',
+      description: 'The write latency being reported',
+      unit: 's',
+      sources: {
+        prometheus: {
+          expr: 'sum(cassandra_keyspace_writelatency_seconds{%(queriesSelector)s} >= 0) by (quantile)',
+        },
+      },
+    },
+
+    readLatencyHeatmap: {
+      name: 'Read latency',
+      nameShort: 'Read latency',
+      type: 'gauge',
+      description: 'The read latency being reported',
+      unit: 's',
+      sources: {
+        prometheus: {
+          expr: 'sum(cassandra_keyspace_readlatency_seconds{%(queriesSelector)s} >= 0) by (quantile)',
+        },
+      },
+    },
```

Comment:
There may be a bug with Signals as the unit doesn't seem to be loading in properly for the heatmap visualisation

Reply:
<img width="958" height="746" alt="Image" src="https://github.com/user-attachments/assets/4e2a26be-71f1-4449-af42-1e57b012732c" />